### PR TITLE
Feature/postgres db engine

### PR DESCRIPTION
**Pull request by evemorgen**

### CODE REVIEW

File: rails_app/config/settings/production.yml [hunk 1]
```diff
@@ -5,14 +5,14 @@ graphite:
     default_get_metric_time: "-14d"
 
 parsers:
-    itaka:
+    www.itaka.pl:
```
Comment (szmo) [inline rails_app/config/settings/production.yml:8]: why this change again?

File: rails_app/config/settings/production.yml [hunk 2]
```diff
@@ -5,14 +5,14 @@ graphite:
...
         number:
             price: "strong[data-js-value='offerPrice']"
         string:
             name: "span[class='productName-holder']"
             country: "span[class='destination-title destination-country-region'] @data-product-country"
             city: "span[class='destination-title destination-country-region'] @data-product-region"
-    x-kom:
+    www.x-kom.pl:
```
Comment (szmo) [inline rails_app/config/settings/production.yml:15]: why this change again?

File: rails_app/config/settings/production.yml (hunk 1 above)
Comment (evemorgen) [reply rails_app/config/settings/production.yml:8]: This is the format that BasicParser takes, just making it up to date with `development.yml`

File: rails_app/config/settings/production.yml (hunk 2 above)
Comment (evemorgen) [reply rails_app/config/settings/production.yml:15]: This is the format that BasicParser takes, just making it up to date with `development.yml`

**Review by szmo — APPROVED** [review]
